src/hooks/useWalletANTs/useWalletANTs.tsx

```diff
@@ -440,6 +442,7 @@ export function useWalletANTs() {
         dispatch: dispatchArNSState,
         emitter: arnsEmitter,
         walletAddress,
+        registryId: arnsContractId,
```

Comment:
I think we've got three different names for the same thing now:

* registryId
* arnsContractId
* ioProcessId

If these are all the same thing, we should settle one a single name.